chingu-coders/Voyage2-Turtles-02
# clocks bug fixes

## Purpose
* Fixes bugs that @ziggysauce cuaght

## Approach/Fixes
* Removes HTML hardcoding of clock settings output values, and dynamically sets input and output values on load

## Testing
* Checkout this PR
* alter clock settings
* reload app and check that range sliders and output numbers persist
